parlai/utils/conversations.py

```diff
@@ -187,21 +257,7 @@ def __next__(self):
 
     def read_conv_idx(self, idx):
         convo = self.conversations[idx]
-        print(BAR)
-
-        high_level = [k for k in convo.keys() if k != 'dialog']
-        if high_level:
-            for key in high_level:
-                print(f'{key}: {convo[key]}')
-            print(SMALL_BAR)
-
-        for act_pair in convo['dialog']:
-            for turn in act_pair:
-                turn_id = turn['id']
-                text = turn['text']
-                print(f'{turn_id}: {text}')
-
-        print(BAR)
+        convo.read()
```

Comment:
i like this a lot. if we were feeling fancy we could also define the `__str__(self)` method for a `Conversation` that would return a string to print when we do `print(convo)`, but I am definitely happy with this

Reply:
That's a good idea... will change it